src/leader.py

```diff
@@ -37,6 +37,7 @@ def __init__(self):
         self._controller = InputController()
 
         db_file = FILE_PATH + "/../" + SUNFOUNDER_PATH + "/config"
+        self._mode = 'normal'
```

Comment:
Why are we using strings for modes?

Reply:
Strings match the codes for the button's on the controller and allow me to map it easier then other data types since I don't have to write extra lines of code to map a button to an integer or another data type.

Reply:
Can't we just say, if we're not in normal mode, run this button's mode (without a series of if elif)? Especially since none of them even take parameters.